scripts/check_public_api_docstrings.py

```diff
@@ -179,11 +250,15 @@ def main():
 
     if not errors:
         logger.info("There are no public API docstring errors.")
+    else:
+        errors_str = (
+            f"\n----- {len(errors)} errors found -----\n"
+            + "\n".join([e.raw_error for e in errors])
+            + "\n----- END -----\n"
+        )
+        logger.error(errors_str)
 
-    for error in errors:
-        logger.error(error)
-
-    assert len(errors) == 0, "There are docstring errors to address."
+    assert len(errors) == 0, f"There are {len(errors)} docstring errors to address."
```

Comment:
Made the output a bit more readable.

Reply:
It's usually better to have logger messages that are a single line. Maybe something like `{len(errors)} errors found: " | ".join([e.raw_error for e in errors])`